src/test/resources/test-fields-in-payload.conf

```diff
@@ -0,0 +1,24 @@
+include "application"
+
```

Comment:
what is the different between this file and the `test-fields-in-payload-payloadText` file?

Probably the other one is missing `payloadWrapperKey`?